src/App.css

```diff
@@ -20,3 +25,25 @@
 .App-link {
   color: #09d3ac;
 }
+
+nav {
+  width: 40%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 2%;
+}
+
+nav a {
+  text-decoration: none;
+  color: black;
+  font-size: 1.1rem;
+  padding: 3% 0;
+  margin: 2%;
+  width: 30%;
```

Comment:
Percents can be super tricky for things like width. If you take a look at the app in mobile view (see the circled sections in the screen shot for how to view as mobile) , notice how the button heights end up different b/c the content of one button has to wrap, while the other one is just fine in smaller screens.

Now, the tough thing here that you may want to get input from the team on is really the question of "How do we want these buttons to look on the smaller screens?" I love the large font personally and would hate to make the text smaller to solve the issue. Should we make the buttons have a fixed width that is large enough to accommodate both texts? Do we want both buttons to be taller? These are questions that I usually reach out to my team's designer about, but a lot of teams don't have designers and in that case, it's perfectly fine and usually best to reach out to other members of the team when making design decisions like this.

![image](https://user-images.githubusercontent.com/34964885/103785442-0afc8a00-4ff0-11eb-909b-635482de39a3.png)

Reply:
Hi @MikeBLambert We've fixed the odd box sizing issue and for now when the screen width is less than 900 the boxes resize to 50px x 50px. The text still wraps though. I'll consult the team as to see how we want to handle the nav buttons on a mobile device.

<img width="343" alt="Screen Shot 2021-01-06 at 7 21 07 PM" src="https://user-images.githubusercontent.com/14856004/103838923-e934ee80-5053-11eb-92ed-437ac1e5c2c9.png">